OpenSim/Tests/Components/testComponents.cpp

```diff
@@ -74,6 +74,8 @@ int main()
         availableComponents.push_back(availableBushingForces[i]);
     }
 
+    // Test PrescribedForce
+    availableComponents.push_back(new PrescribedForce());
```

Comment:
I think this is a memory leak, since `availableComponents` doesn't own its objects (https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/Object.h#L460). This might be a fix:

```
std::unique_ptr<PrescribedForce> f(new PrescribedForce());
availableComponents.push_back(f.get())
```